file-attribute-caching/src/test/kotlin/com/pkware/file/fileAttributeCaching/ExpirableCacheTest.kt

```diff
@@ -0,0 +1,106 @@
+package com.pkware.file.fileAttributeCaching
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.util.concurrent.TimeUnit
+
+class ExpirableCacheTest {
+
+    private val attributeCache = ExpirableCache<String, String>(
+        TimeUnit.SECONDS.toMillis(1)
```

Comment:
Id rather make the tests run in parallel here than reduce the cache expiration time any further. All of the tests run in under 10 seconds so that's not a deal breaker/huge concern with me.

If you can provide insight on how to run the test suite in parallel (i thought this was already happening by default but its not) that would be appreciated

Reply:
they are currently run in series when i thought they would be parallelized.